core/src/main/scala/io/finch/ToService.scala

```diff
@@ -117,8 +118,8 @@ object ToService {
 
             Trace.captureIfNeeded(trc)
 
-            val accept =
-              if (opts.negotiateContentType) req.accept.map(a => Accept.fromString(a)) else Nil
+            val negotiateContent = isNegotiable.fold(_ => true, _ => false)
```

Comment:
I think we can move this out of `apply` (request path) and reuse the value instead of recomputing it.